mg.js

```diff
@@ -22,51 +16,147 @@ function shuffleArray(array) {
   return array;
 }
 
+function startTimer() {
+  startTime = Date.now();
+  timerInterval = setInterval(() => {
+    const elapsedTime = Math.floor((Date.now() - startTime) / 1000);
+    document.getElementById("timer").textContent = `Time: ${elapsedTime}s`;
+  }, 1000);
+}
+
+function stopTimer() {
+  clearInterval(timerInterval);
+}
+
+function updateFlipCounter() {
+  flipCount++;
+  document.getElementById("flip-counter").textContent = `Flips: ${flipCount}`;
+}
+
 function createCardButton(frontImg) {
   const cardButton = document.createElement("button");
   cardButton.className = "card";
-
   const img = document.createElement("img");
   img.src = backImg;
   cardButton.appendChild(img);
-  let showFront = false;
 
   cardButton.addEventListener("click", () => {
-    if (!startTime) {
-      startTime = Date.now();
-      timerInterval = setInterval(() => {
-        const elapsedTime = parseInt((Date.now() - startTime) / 1000);
-        console.log("Elapsed time: " + elapsedTime + " seconds");
-      }, 1000);
+    if (!timerStarted) {
+      timerStarted = true;
```

Comment:
I think this boolean could be set by the function `startTimer`. It feels more like it is its responsibility to do it